index.html

```diff
@@ -122,12 +122,12 @@ <h1>Tonic</h1>
       </div>
       
       <div class="card_info">
-        <div class="card_title">
+        <div class="card_title" id="title_2">
           <h1>Multi-Post
             Stories</h1>
         </div>
 
```

Comment:
- [ ] The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.

Reply:
This information is not linked to the popUp

Reply:
This is being displayed as the projects. The array of objects is what you are using for the modal popup.
Both the projects display as a whole and the modal pop-ups for each should be populated from the array of objects.